### ISSUE TITLE
Man-in-the-Middle on Vanilla Switch?

### ISSUE THREAD

**Issue by MontyMole98** [kinnay/NintendoClients#119]
Hi, I was just curious if you could retrieve the user ID and password from a vanilla switch using mitm. Dumb question, I know, but I can't find a better place to ask.

**Comment by kinnay** [comment]
MITM is only possible on a hacked Switch. The reason is that the Switch uses TLS for all requests. To do a MITM, you first have to [disable CA verification](https://github.com/misson20000/exefs_patches) on your Switch.

I'm not sure what you mean by 'vanilla switch', but there is no way to retrieve the user id and password without [atmosphere](https://github.com/Atmosphere-NX/Atmosphere).

**Comment by K4CZP3R** [comment]
Yup, this helps. First option is not possible for me because of banned switch. Will def. try the "system save data"one!

**Comment by nicholasyoannou** [comment]
Is it possible to enable SSL proxying for all of Nintendo's URLs? I'm interested to see what happens behind the scenes, but I'm unsure about enabling it especially for Switch Online.
